index.js

```diff
@@ -259,6 +260,31 @@ const retryAndBackoff = async (fn, isRetryable, retries = 0, maxRetries = 12, ba
   }
 }
 
+function configureProxy(inputIgnore, inputProxy) {
```

Comment:
Nitpicky, but I'm not keen on these variable names. I'd prefer they be named how they are when you pass them in